docker/puppet-agent-alpine/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN apk add --no-cache --update-cache --repository http://nl.alpinelinux.org/alp
     apk add --no-cache cmake boost-dev make curl git curl-dev ruby ruby-dev yaml-cpp-dev jq openjdk8 augeas ruby-augeas ruby-etc ruby-json ruby-multi_json libressl-dev virt-what && \
     gem install --no-rdoc --no-ri deep_merge semantic_puppet puppet-resource_api locale httpclient fast_gettext concurrent-ruby thor hocon
 
-RUN mkdir /workspace && \
+RUN mkdir -p /workspace && \
```

Comment:
Same thing about a `-p` for one level deep?

Reply:
same answer here https://github.com/puppetlabs/r10k/pull/1108#discussion_r545250666